Sources/TuistKit/Utils/InfoPlistProvisioner.swift

```diff
@@ -88,12 +89,19 @@ class InfoPlistProvisioner: InfoPlistProvisioning {
                 "UIInterfaceOrientationLandscapeLeft",
                 "UIInterfaceOrientationLandscapeRight",
             ]
+            base["UIMainStoryboardFile"] = "$(PRODUCT_NAME)"
```

Comment:
We are assuming here that someone else in the code base is going to generate the storyboard with the name of the product. The downside that I see in this approach is that if someone changes the name of the storyboard in the generator the integration between the project and the storyboard will get broken. I'd suggest to have a variable/method that returns the storyboard name for a given target and then call the method from the generator and from here.
Alternatively, we could have a test that verifies that the Info.plist's manifest points to a Storyboard that exists but I think that'd be more tedious.

Reply:
Yer I had this worry in my thoughts but haven't looked to solve it yet. Maybe by adding some configuration in the `Project.swift` might solve this issue as well as understanding what storyboards to look for.